src/libraries/Common/src/Interop/Windows/Netapi32/Interop.DS_DOMAIN_TRUSTS.cs

```diff
@@ -0,0 +1,24 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class Netapi32
+    {
+        [StructLayout(LayoutKind.Sequential)]
+        internal sealed class DS_DOMAIN_TRUSTS
```

Comment:
```suggestion
        internal struct DS_DOMAIN_TRUSTS
```

Reply:
Done in 235ba378caccd739fd10653a24f739a5321edae8